src/ui/common/src/components/pages/workflows/components/MetricItem.tsx

```diff
@@ -98,20 +98,24 @@ const MetricItem: React.FC<MetricItemProps> = ({ metrics }) => {
     setExpanded(!expanded);
   };
 
+  // 0.875rem is the size of the ShowMore.
+  // Add this additional padding if we don't have more than 1 metric to keep the rows equal size.
   return (
     <Box>
       {metrics.length > 0 ? (
-        <>
+        <Box sx={metrics.length === 1 && { padding: '0.875rem 0 0.875rem 0' }}>
           {metricList}
           <ShowMore
             totalItems={metrics.length}
             numPreviewItems={metricsToShow}
             expanded={expanded}
             onClick={toggleExpanded}
           />
-        </>
+        </Box>
       ) : (
-        <Typography variant="body1">No metrics.</Typography>
+        <Typography sx={{ padding: '0.875rem 0 0.875rem 0' }} variant="body1">
```

Comment:
same as above. You can also factor out the padding into it's own variable too. So two lines: one for the const showMoreFontSize and one for the backtick string (probably to be named showMorePadding)